macros/edr/dbt_artifacts/upload_dbt_exposures.sql

```diff
@@ -1,8 +1,9 @@
 {%- macro upload_dbt_exposures() -%}
+    {% set relation = elementary.get_elementary_relation('dbt_exposures') %}
     {% set edr_cli_run = elementary.get_config_var('edr_cli_run') %}
-    {% if execute and not edr_cli_run %}
+    {% if execute and relation and not edr_cli_run %}
         {% set exposures = graph.exposures.values() | selectattr('resource_type', '==', 'exposure') %}
-        {% do elementary.upload_artifacts_to_table(this, exposures, elementary.get_flatten_exposure_callback()) %}
+        {% do elementary.upload_artifacts_to_table(relation, exposures, elementary.flatten_exposure, should_commit=true) %}
```

Comment:
I think that you should pass the should_commit=True only if it's being called from the on-run-end